models/engine/file_storage.py

```diff
@@ -0,0 +1,61 @@
+"""
+    This module handles serialization/deserialization of objects
+"""
+import json
+from pathlib import Path
+
+class FileStorage:
+    """
+    Serializes instances to a JSON file and deserializes
+    JSON file to instances
+
+    """
+    def __init__(self, file_path, objects):
+        """Instantiates the class with prameters"""
+        self.file_path = file_path
+        self.objects = objects
+
+    @property
+    def file_path(self):
+        """Get/set the file path for the JSON object"""
+        return (self.__file_path)
+
+    @file_path.setter
+    def file_path(self):
+        if not self.__path:
+            self.__path = str(Path.cwd() / objects.json)
+    
+    def objects(self):
+        """Get/set the dictionary that stores all objects by <class name>.id"""
+        return (self.__objects)
+
+    @objects.setter
+    def objects(self):
+        try:
+            if self.__objects:
+                pass
+        except (NameError):
+            self.__objects = {}
+
+    def all(self):
+        """Returns the dictionary __objects"""
+        self.objects()
```

Comment:
This method is missing the "return" keyword, is this intentional?

Reply:
Yeah. That was because of the object setter method

Reply:
oh, I see it now. You won't need the object setter after making __objects a class attribute, you'll just return self.__object directly from the all() method